apis/scheduling/v1alpha1/reservation_types.go

```diff
@@ -100,6 +125,9 @@ const (
 	// ReservationExpired indicates the Reservation is expired, which the object is not available to allocate and will
 	// get cleaned in the future.
 	ReservationExpired ReservationPhase = "Expired"
+
+	ReasonReservationAvailable = "Available"
+	ReasonReservationExpired   = "Expired"
```

Comment:
move these two constants into another group of const?

Reply:
done